instrumentation/opentelemetry-instrumentation-falcon/src/opentelemetry/instrumentation/falcon/__init__.py

```diff
@@ -524,9 +524,7 @@ def process_response(self, req, resp, resource, req_succeeded=None):  # pylint:d
             if exc_type and not req_succeeded:
                 if "HTTPNotFound" in exc_type.__name__:
                     status = "404"
-                elif isinstance(exc, falcon.HTTPError) or isinstance(
-                    exc, falcon.HTTPStatus
-                ):
+                elif isinstance(exc, falcon.HTTPError, falcon.HTTPStatus):
```

Comment:
```suggestion
                elif isinstance(exc, (falcon.HTTPError, falcon.HTTPStatus)):
```

isinstance has only two arguments :)